runChecks.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python
+
+"""
+Replacement for test-math-dependencies target in Makefile.
+
+"""
+
+from __future__ import print_function
+import os
+import os.path
+import sys
+import re
+import glob
+
+winsfx = ".exe"
+testsfx = "_test.cpp"
+def files_in_folder(folder):
+    files = []
+    for f in glob.glob(folder):
+        if os.path.isdir(f):
+            files += files_in_folder(f+os.sep+"**")
+        else:
+            files.append(f)
+    return files
```

Comment:
Maybe `os.walk` could be better fit to get all the files (including subfolders).

```
def files_in_folder(folder):
    files = []
    for root, _, filenames in os.walk(folder):
        for filename in filenames:
            files.append(os.path.join(root, filename))
    return files
```

And for only the main folder

```
def files_in_folder(folder):
    files = []
    for root, _, filenames in os.walk(folder):
        for filename in filenames:
            files.append(os.path.join(root, filename))
        break
    return files
```

Reply:
I tried using `os.walk` but  it does not support search with "wildcards", meaning we cant list file that match `'stan/math/*/*/meta'`.